orangecontrib/associate/widgets/__init__.py

```diff
@@ -6,3 +6,18 @@
 # Background color for category background in menu
 # and widget icon background in workflow.
 BACKGROUND = "#cccc66"
+
+# Location of widget help files.
+WIDGET_HELP_PATH = (
```

Comment:
I already wrote [this](https://github.com/biolab/orange3/wiki/Building-documentation-for-add-ons), but I'll update the entire example add-on in the nearby future.